### PR TITLE
BUGFIX:TMA-707 - Write log for creating and deleting client when proisioning

### PR DESCRIPTION
**Pull request by sangtm**
…

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/660/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/623/ : SUCCESS in 22m 12s

File: lib/gooddata/models/data_product.rb [hunk 1]
```diff
@@ -125,7 +125,10 @@ def update_clients(data, options = {})
       data = GoodData::Helpers.get_path(res, ['updateClientsResponse'])
       if data
         result = data.flat_map { |k, v| v.map { |h| GoodData::Helpers.symbolize_keys(h.merge('type' => k)) } }
-        result.select { |r| r[:status] == 'DELETED' }.peach { |r| r[:originalProject] && client.delete(r[:originalProject]) } if options[:delete_projects]
+        result.select { |r| r[:status] == 'DELETED' }.peach do |r|
+          r[:originalProject] && client.delete(r[:originalProject])
+          GoodData.logger.info("Deleted client: #{r[:id]}; link: #{r[:client]}; originalProject: #{r[:originalProject]}")
```
Comment (danh-ung) [inline lib/gooddata/models/data_product.rb:130]: log originalProject --> project

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: lib/gooddata/models/data_product.rb (hunk 1 above)
Comment (sangtm) [reply lib/gooddata/models/data_product.rb]: Updated it. Thank you

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/662/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/625/ : SUCCESS in 21m 32s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/669/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/628/ : SUCCESS in 21m 54s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by sangtm** [comment]
ok to test

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/677/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/636/ : FAILURE in 20m 07s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/678/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/637/ : SUCCESS in 18m 03s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/679/ : SUCCESS in 7s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/638/ : SUCCESS in 21m 47s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/686/ : SUCCESS in 7s

**Comment by sangtm** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/687/ : SUCCESS in 7s

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ruby-gate-PR1753

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-sonarqube-scan-zuul-el7/41/ : SUCCESS in 3m 10s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/524/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-fossa-scanning-zuul-el7/80/ : SUCCESS in 1m 14s (non-voting)

**Pull request merged**